exoscale/resource_exoscale_database_pg.go

```diff
@@ -414,7 +423,12 @@ func resourceDatabaseApplyPg(ctx context.Context, d *schema.ResourceData, client
 	}
 
 	if v := databaseService.Version; v != nil {
-		pg[resDatabaseAttrPgVersion] = *v
+		pg[resDatabaseAttrPgActualVersion] = *v
+	}
+
+	if v := databaseService.Version; v != nil {
+		version, _ := semver.NewVersion(*v)
+		pg[resDatabaseAttrPgVersion] = strconv.FormatUint(version.Major(), 10)
 	}
```

Comment:
I would suggest this instead, this is simple enough and avoids an additional dependency.

```suggestion
	if databaseService.Version != nil {
		pg[resDatabaseAttrPgVersion] = strings.SplitN(*databaseService.Version, ".", 2)[0]
	}
```

Also, do you think `actual_version` is useful? Would you use it?

Reply:
I agree with your suggestion regarding simplification and avoiding redundant dependency.
However I do think that `actual_version` may be useful in case of tracking changes applied externally especially from the infosec perspective.
It would be more useful if I could set exact version indeed, but it is another story I guess :)